CenterForOpenScience/osf.io
# Hotfix/autoban ip block [hotfix]

## Purpose

We should go ahead and nuke all users that are IP blocked.

## Changes

Auto ban all IP blocked users

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk? Some users who are legitimate might get ip blocked.

Any concerns/considerations/questions that development raised? N/A

## Documentation
N/A

## Side Effects

N/A
## Ticket

N/A
